lib/Adapter/Php/InMemory/InMemoryQuery.php

```diff
@@ -47,4 +48,9 @@ public function file(string $path): ?FileRecord
     {
         return null;
     }
+
+    public function member(string $name): ?MemberRecord
+    {
+        return null;
```

Comment:
Should ultimately remove these "test" implementations, it's just as easy to use the "real" ones.